content/applications/productivity/calendar/outlook.rst

```diff
@@ -56,7 +56,7 @@ Finally, click :guilabel:`Add` to generate the client secret (:guilabel:`Secret
 Configuration in Odoo
 =====================
 
-In the Odoo database, go to :menuselection:`Settings --> General Settings --> Integrations` and
+In the Odoo database, go to :menuselection:`Settings app --> Calendar ` in the left menu and
 activate the :guilabel:`Outlook Calendar` setting.
```

Comment:
Let's remove the extra space in the **menuselection tag**.

Doing so then allows word "activate" to move up a line since it's now within the [100 character line limit](https://www.odoo.com/documentation/17.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character).
```suggestion
In the Odoo database, go to :menuselection:`Settings app --> Calendar` in the left menu and activate
the :guilabel:`Outlook Calendar` setting.
```